### DISCUSSION TITLE
RE: Arcticons.com Domain - HTTPS Issue

### DISCUSSION THREAD

**Discussion by aaofyi** [Arcticons-Team/Arcticons#1210] [General]
Hey Donnno, I noticed the domain was throwing the "SSL_ERROR_UNRECOGNIZED_NAME_ALERT". Just figured I'd let you know this was my fault because I had HSTS enabled on the domain when it was in my account. It was set for 6 months when I did it, so hopefully that problem will clear up for all users after that clock runs out; or HSTS could be re-enabled but that is dependent on your hosting provider.

Just wanted to let you know, sorry bout that!

**Comment by Donnnno** [comment]
Hi, late reply, I know. This always bothered me, but I just wasn't able to fix this with my hosting company. I've set up a redirect.pizza forward, so it should be fixed now!
